### PR TITLE
Remove sample channel from `ramenctl config`

### PR DESCRIPTION
**Pull request by nirs**
The channel is part of the application, not a ramen configuration, so it is installed
now by test/basic-test/deploy.

### CODE REVIEW

**Comment by nirs** [comment]
It does not work yet, since we don't install the channel from the busybox app.

**Comment by nirs** [comment]
Testing with https://github.com/RamenDR/ocm-ramen-samples/pull/31

```
$ test/basic-test/deploy dr1 -v
2023-06-27 23:37:03,574 DEBUG   [deploy] Parsed arguments: Namespace(verbose=True, cluster='dr1')
2023-06-27 23:37:03,574 INFO    [deploy] Creating temporary directory /tmp/ramen-test/basic-test
2023-06-27 23:37:03,574 INFO    [deploy] Cloning ocm-ramen-samples
2023-06-27 23:37:04,353 INFO    [deploy] Creating kustomization for using cluster 'dr1'
2023-06-27 23:37:04,353 INFO    [deploy] Deploying busybox example application
2023-06-27 23:37:04,667 DEBUG   [deploy] namespace/busybox-sample created
2023-06-27 23:37:04,678 DEBUG   [deploy] namespace/ramen-samples created
2023-06-27 23:37:04,696 DEBUG   [deploy] channel.apps.open-cluster-management.io/ramen-gitops created
2023-06-27 23:37:04,713 DEBUG   [deploy] placementrule.apps.open-cluster-management.io/busybox-placement created
2023-06-27 23:37:04,730 DEBUG   [deploy] subscription.apps.open-cluster-management.io/busybox-sub created
2023-06-27 23:37:04,740 DEBUG   [deploy] drplacementcontrol.ramendr.openshift.io/busybox-drpc created
2023-06-27 23:37:04,747 INFO    [deploy] waiting for namespace busybox-sample
2023-06-27 23:37:04,808 DEBUG   [deploy] namespace/busybox-sample exists
2023-06-27 23:37:04,808 INFO    [deploy] Waiting until busybox drpc reports phase
2023-06-27 23:37:05,913 DEBUG   [deploy] drpc/busybox-drpc exists
2023-06-27 23:37:05,913 INFO    [deploy] Waiting until busybox drpc is deployed
2023-06-27 23:37:06,062 DEBUG   [deploy] drplacementcontrol.ramendr.openshift.io/busybox-drpc condition met
2023-06-27 23:37:06,064 INFO    [deploy] Waiting until application is replicated
2023-06-27 23:38:34,708 DEBUG   [deploy] drpc/busybox-drpc exists
2023-06-27 23:38:34,708 INFO    [deploy] Application was deployed successfully
```

File: test/addons/csi-addons/start [hunk 1]
```diff
@@ -9,8 +9,9 @@ import sys
 import drenv
 from drenv import kubectl
 
-# Using main till a release is available with lastSyncTime.
-VERSION = "main"
+# We don't have a release with lastSyncTime, and main is currently broken (no
+# quay.io image matching latest code).
+VERSION = "ee9403c55c3aee9da7b25785b87e370c4095d881"
```
Comment (ELENAGER) [inline test/addons/csi-addons/start:14]: It comes from another PR?

Comment (nirs) [reply test/addons/csi-addons/start:14]: This uses the last commit that has a successful build and an image in quay.io:
https://github.com/csi-addons/kubernetes-csi-addons/commit/ee9403c55c3aee9da7b25785b87e370c4095d881

Sine the issue was fixed upstream we don't this workaround now.

**Review by ELENAGER — APPROVED** [review]

**Review by Shwetha-Acharya — APPROVED** [review]

**Pull request merged**